.gitignore

```diff
@@ -98,6 +98,13 @@ ipch/
 *.cachefile
```

Comment:
The changes made to .gitignore mostly seem to be double entries, better drop the changes to .gitignore.

Reply:
sorry for that, i revert .gitignore to original state https://www.gitignore.io/api/visualstudio,visualstudiocode